ada/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+class RedirectUrl(models.Model):
+
+    content = models.TextField(
+        default="This entry has no copy",
+        help_text="The content of the mesage that this link was sent in",
+    )
+    symptom_check_url = models.URLField(
+        max_length=200, blank=False, default="http://symptomcheck.co.za"
+    )
+    parameter = models.IntegerField(null=True)
+    time_stamp = models.DateTimeField(auto_now=True)
+
+    def my_counter(self):
+        total_number = RedirectUrlsEntry.objects.filter(
+            symptom_check_url=self.id
+        ).count()
+        return total_number
+
+    def __str__(self):
+        if self.symptom_check_url:
+            return (
+                f"{self.parameter}: https://hub.momconnect.co.za/redirect/{self.id} \n"
```

Comment:
It seems odd that `hub.momconnect.co.za` is hardcoded here... In QA this will show the production URL. But we also don't have the `request` here to create a full URL. 

There is a `get_absolute_url` method on the model, which if included, django admin will create a `View on site` button with the URL. That seems like the more appropriate method to use https://docs.djangoproject.com/en/3.1/ref/models/instances/#django.db.models.Model.get_absolute_url